### PR TITLE
Example: Elmish.HTML Hello World

### PR DESCRIPTION
**Pull request by gasi**
image

### CODE REVIEW

File: counter/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Counter
```
Comment (fsoikin) [inline counter/README.md]: I really wanted to call it a "counterexample" :-)
Plus, I think a small explanation of the intent would help.

```suggestion
# A counterexample
A set of progressively more complex examples, demonstrating ways to compose complex UIs out of simpler ones, using an inc/dec counter as base.
```

**Review by fsoikin — APPROVED** [review]
Looks solid, nice and clean 👍

**Comment by gasi** [comment]
@fsoikin Looks like either you have to merge it or add me to the list of contributors in this repo.

**Comment by fsoikin** [comment]
Added you

**Pull request merged**